### ISSUE TITLE
Verify C++ inference runtime

### ISSUE THREAD

**Issue by iizukak** [blue-oil/blueoil#1020]
Currently, we have C++ inference code on `runtime` directory.
https://github.com/blue-oil/blueoil/tree/master/runtime

We are not sure that runtime can run correctly.

- [x] Build for classification
- [x] Check inference
- [x] Build for object detection
- [x] Check inference
- [x] Update README.md

**Comment by oatawa1** [comment]
I checked inference results in classification task, and I think the class label order maybe **invert**.
The input image is cat.
From class position of cat is in 4th.
But the inference result from softmax shows the highest score in 6th (which is maybe 10th-4th).

Results are here.
```
./run -i cat.npy -c model_classification/meta.yaml
classes:
airplane
automobile
bird
cat
deer
dog
frog
horse
ship
truck
task: IMAGE.CLASSIFICATION
expected input shape:
1
32
32
3
Run
Results !
shape:1 10
0.000527386 0.000344286 0.14302 0.0831289 0.0965507 0.00173161 0.673597 0.000440438 0.000252013 0.00040764
```

**Comment by oatawa1** [comment]
PR #1030 update the README.md

**Comment by oatawa1** [comment]
`PerImageStandardization` is not compatible with the current runtime.

I found that PerImageStandardization is already implemented in blueoil_data_processor.cpp but it’s not in mapping process in blueoil.cpp

Results in `PerImageStandardization is not register method:`

**Comment by oatawa1** [comment]
Readme now updated in#1030 is merged. 
The problem about PerImageStandardization is solved in #1031 and merged.

So I close this issue.
